CGPs/0036/staging.json

```diff
@@ -0,0 +1,152 @@
+[
+    {
+      "contract": "Freezer",
+      "function": "freeze",
+      "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C"], // exchangeEUR address
+      "value": "0"
+    },
+    {
+        "contract": "Freezer",
+        "function": "freeze",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0"], // stableTokenEUR address
+        "value": "0"
+      },
+
+
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x4D169C0Ec17D3dEcB250701Aa64e966bE98a696B"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x034217fB0C08dec29a6beA99e75aAC39baf7459A"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0xD20f803a1489849031725479Df27741c3326eD15"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x26412103865Be3F80152EA10E61E6294ed0e0DC9"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x7Ba21D1f769Ae9feCF58dEfB5bD08905Bd3B707B"], // stableTokenEUR address
+        "value": "0"
+      },
+
+
+      // fund the oracles?
```

Comment:
how would the governance proposal would work to send CELO from the community fund to each oracle?
If risky or unnecessary, can be removed and we test manually

Reply:
You just need the destination to be the oracle address, the function to be empty, the args to be empty, and the value to be whatever value you want to fund the oracle address with. And then one such tx for each oracle being funded